Hybrid studies : bounding unsup energy variable from modeler [ANT-4033] (#3382)
AntaresSimulatorTeam/Antares_Simulator: docs/user-guide/solver/08-hybrid-studies.md

   fields:
      - id: flow
      - id: angle
	  - id: to-area-bound
      - id: from-area-bound
   area-connection:

>       - injection-field: flow

Reviewer: Remove the dashes

Author: We can't do this yet. It will be the subject of a [further ticket](https://gopro-tickets.rte-france.com/browse/ANT-4481).

   fields:
      - id: flow
      - id: angle

> 	  - id: to-area-bound

Reviewer 2: issue: replace with spillage-bound/unsupplied-energy-bound

Author: Was left like this on purpose : port fields does not have to be named as you suggest, even if section **area-connection** fields have to be compliant with this.

      - id: from-area-bound
   area-connection:
      - injection-field: flow
	  - spillage-bound: to-area-bound
      - unsupplied-energy-bound: from-area-bound
~~~

**area-connection** is the name of the optional section to use. It is mandatory if you want to use such a port type to 
connect modeler components to solver areas.
Here we describe the role of each field contained in **area-connection**. The value for each of them
must be an existing port name in the same port type.
All components connected to this port and area (via an **area-connections** system section entry, 
see previous paragraph [Defining the connections](#defining-the-connections)) will have their corresponding linear 
expression contributing to a constraint of the linear problem.
The nature of this contribution depends on the field : 
  - **injection-field**: the linear expression is injected in the balance constraint of the area.

> 	This is done with respect to a convention (see next [Optimization model](#optimization-model)).

Reviewer: ```suggestion
	*Production* should be counted positively ! See next [Optimization model](#optimization-model) for more details.
```

Author: Done

	
  - **spillage-bound**: the linear expression is added to the sum of all variables or linear expressions already used 

>     to bound the spillage in the constraint called "fictitious load".

Author: Same as above.

>   - **unsupplied-energy-bound** : the linear expression is added to any linear expression already used to bound the unsupplied energy.

Reviewer: ```suggestion
  - **unsupplied-energy-bound** : the linear expression is added to any linear expression already used to bound the unsupplied energy. *Loads* should be counted positively ! See next [Optimization model](#optimization-model) for more details.
```

Author: Same as above.

Author: Done

#### Adding a linear expreesion in optimization model

For more details about how to add a contribution of a component from GEMS to the leagcy linear problem, and 

> which sign (positive / negative) to give them in the constraint, please downlaod the Antares repository and read file :

Reviewer 2: ```suggestion
which sign (positive / negative) to give them in the constraint, please download the Antares repository and read file :
```

Author: A link to a source file was added instead

> **docs/Architecture_Decision_Records/from-GEMS-to-legacy-linear-preblem.md**

Reviewer 2: suggestion: maybe provide a direct lik to the file and avoid mentionning downloading the source code

Author: Done
